sources/presentation/Stride.Core.Presentation/Controls/NumericTextBox.cs

```diff
@@ -413,6 +413,38 @@ private void UpdateValue(double value)
             }
         }
 
+        /// <summary>
+        /// tries to parse the value of the textbox into a double, accommodating various cultural settings and preferences
+        /// </summary>
+        /// <param name="value">text value of the textbox</param>
+        /// <param name="result">the resulting numeric value if parsing is successful</param>
+        /// <returns>whether parsing the value was successful</returns>
+        private static bool TryParseValue(string value, out double result)
+        {
+            //thousands are disallowed as they might lead to decimal seperators falsely being interpreted as thousands
+            const NumberStyles numberStyle = NumberStyles.Any & ~NumberStyles.AllowThousands;
```

Comment:
This is a breaking change (at least visually) since we were using thousands separators previously for a better display.